usefulUtilities/sit/sitServer.js

```diff
@@ -0,0 +1,224 @@
+//
+// sitServer.js
+//
+// Created by Robin Wilson 1/17/2019
+//
+// Copyright 2017 High Fidelity, Inc.
+//
+// Distributed under the Apache License, Version 2.0.
+// See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+// 
+// sitServer.js is the gate to allow an avatar to sit. Multiple avatars can not sit in the same chair.
+// Spawns the canSitZone per chair
```

Comment:
not accurate comment anymore since we removed the zones

Reply:
ack